Mathlib/RingTheory/TensorProduct/Basic.lean

```diff
@@ -948,6 +948,51 @@ theorem congr_trans [Algebra S C] [IsScalarTower R S C]
 
 theorem congr_symm (f : A ≃ₐ[S] B) (g : C ≃ₐ[R] D) : congr f.symm g.symm = (congr f g).symm := rfl
 
+variable (R A B C) in
+/-- Tensor product of algebras analogue of `mul_left_comm`. -/
```

Comment:
```suggestion
/-- Tensor product of algebras analogue of `mul_left_comm`.

This is the algebra version of `TensorProduct.leftComm`. -/
```
and same for tensorTensorTensorComm` below.

Reply:
Thanks, done